src/utils/theme.tsx

```diff
@@ -790,48 +794,6 @@ export function getThemedComponents(theme: Theme) {
           },
         },
       },
-      MuiToggleButtonGroup: {
```

Comment:
Some context on this change:

These global styles to the ToggleButton made it difficult to do any one-off customizations of this component, which we needed for the graph time range selector. I removed these styles, and made a re-usable styled component that we can use throughout the app for existing ToggleButtons. Then we could make the necessary modifications in the GraphTimeRangeSelector component.